Gravity-tube/package.json

```diff
@@ -15,6 +15,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-icons": "^4.3.1",
+    "react-player": "^2.10.0",
```

Comment:
instead of this can use video.js as that will give you better performance, will be hard to setup in comparison to this but a better solution